### PR TITLE
Remove unnecessary files

### PR DESCRIPTION
**Pull request by gen0083**
## Issue
- These files are not needed under version control, probably.

## Overview (Required)
-

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by konifar** [comment]
Good catch! Thanks. LGTM 👍

File: app/src/main/java/io/github/droidkaigi/confsched2017/receiver/NotificationReceiver.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ private void showChildNotification(Context context, Intent intent, boolean heads
         int priority = headsUp ? NotificationCompat.PRIORITY_HIGH : NotificationCompat.PRIORITY_DEFAULT;
         Intent openIntent = SessionDetailActivity.createIntent(context, sessionId);
         openIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);
-        PendingIntent pendingIntent = PendingIntent.getActivity(context, 0, openIntent, 0);
+        PendingIntent pendingIntent = PendingIntent.getActivity(context, 0, openIntent, PendingIntent.FLAG_UPDATE_CURRENT);
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/receiver/NotificationReceiver.java]: Is it okay to merge this on this PR?
Or will you revert this change and send another PR?

**Comment by gen0083** [comment]
sorry, I mistake to push without switching branch...

File: app/src/main/java/io/github/droidkaigi/confsched2017/receiver/NotificationReceiver.java (hunk 1 above)
Comment (gen0083) [inline app/src/main/java/io/github/droidkaigi/confsched2017/receiver/NotificationReceiver.java]: I'll revert it

**Comment by konifar** [comment]
No problem! Will you revert the commit?

**Comment by gen0083** [comment]
done it, thanks!

**Comment by konifar** [comment]
Thanks! I'll wait CI and merge.
